data/i18n/hi/hi.toml

```diff
@@ -1,195 +1,195 @@
 
 # Labels for the docs portal home page.
 [docs_label_browse]
-other = "Browse Docs"
+other = "दस्तावेज़ ब्राउज़ करें"
 
 [docs_label_contributors]
-other = "Contributors"
+other = "योगदानकर्ता"
 
 [docs_label_users]
-other = "Users"
+other = "उपयोगकर्ता"
 
 [docs_label_i_am]
-other = "I AM..."
+other = "मैं हूँ..."
 
 # layouts > blog > pager
 
 [layouts_blog_pager_prev]
-other = "<< Prev"
+other = "<< पिछला"
 
 [layouts_blog_pager_next]
-other = "Next >>"
+other = "अगला >>"
 
 # layouts > blog > list
 
 [layouts_case_studies_list_tell]
-other = "Tell your story"
+other = "अपनी कहानी बताओ"
 
 # layouts > docs > glossary
 
 [layouts_docs_glossary_description]
-other = "This glossary is intended to be a comprehensive, standardized list of Kubernetes terminology. It includes technical terms that are specific to K8s, as well as more general terms that provide useful context."
+other = "इस शब्दावली का उद्देश्य कुबेरनेट्स शब्दावली की एक व्यापक, मानकीकृत सूची है। इसमें तकनीकी शब्द शामिल हैं जो K8s के लिए विशिष्ट हैं, साथ ही अधिक सामान्य शब्द जो उपयोगी संदर्भ प्रदान करते हैं।"
 
 [layouts_docs_glossary_filter]
-other = "Filter terms according to their tags"
+other = "शब्दों को उनके टैग के अनुसार फ़िल्टर करें"
 
 [layouts_docs_glossary_select_all]
-other = "Select all"
+other = "सभी का चयन करे"
 
 [layouts_docs_glossary_deselect_all]
-other = "Deselect all"
+other = "सभी को अचयनित करें"
 
 [layouts_docs_glossary_aka]
-other = "Also known as"
+other = "के रूप में भी जाना जाता है"
 
 [layouts_docs_glossary_click_details_before]
-other = "Click on the"
+other = "पर क्लिक करें"
 
 [layouts_docs_glossary_click_details_after]
-other = "indicators below to get a longer explanation for any particular term."
+other = "किसी विशेष शब्द के लिए लंबी व्याख्या प्राप्त करने के लिए नीचे दिए गए संकेतक।"
 
 # layouts > docs > search
 
 [layouts_docs_search_fetching]
-other = "Fetching results.."
+other = "परिणाम लाए जा रहे हैं.."
 
 # layouts > partial > feedback
 
 [layouts_docs_partials_feedback_thanks]
-other = "Thanks for the feedback. If you have a specific, answerable question about how to use Kubernetes, ask it on"
+other = "प्रतिक्रिया के लिए धन्यवाद। यदि आपके पास कुबेरनेट्स का उपयोग करने के तरीके के बारे में कोई विशिष्ट, जवाबदेह प्रश्न है, तो इस पर पूछें"
 
 [layouts_docs_partials_feedback_issue]
-other = "Open an issue in the GitHub repo if you want to "
+other = "यदि आप चाहते हैं तो गिटहब रेपो में एक इशू खोलें "
 
 [layouts_docs_partials_feedback_problem]
-other = "report a problem"
+other = "समस्या के बारे में बताएं"
 
 [layouts_docs_partials_feedback_or]
-other = "or"
+other = "या"
 
 [layouts_docs_partials_feedback_improvement]
-other = "suggest an improvement"
+other = "सुधार का सुझाव दें"
 
 # Community links
 [community_twitter_name]
-other = "Twitter"
+other = "ट्विटर"
```

Comment:
```suggestion
other = "Twitter"
```

Reply:
These terms are Trademarks of their respective corporations. We should keep them as is